DataFormats/TotemRawData/interface/TotemStructuralVFATId.h

```diff
@@ -0,0 +1,56 @@
+/****************************************************************************
+*
+* This is a part of TOTEM offline software.
+* Authors: 
+*   Jan Kašpar (jan.kaspar@gmail.com)
+*    
+****************************************************************************/
+
+#ifndef DataFormats_TotemRawData_TotemStructuralVFATId
+#define DataFormats_TotemRawData_TotemStructuralVFATId
+
+#include <cstdint>
+
+/**
+ * A read-out chip (VFAT) described by its detector and its position within the detector.
+ *
+ * detId can be interpreted by passing it to constructor of a class inheriting from DetId, e.g. TotemRPDetId.
+ *
+ * For RP data VFATs, chipPosition is 0 to 3.
+ * For RP CC VFATs, detId corresponds to the first plane and chipPosition=100.
+ *
+ */
+class TotemStructuralVFATId
+{
+  public:
+    TotemStructuralVFATId(uint32_t di=0, uint32_t cp=0) : detId(di), chipPosition(cp)
+    {
+    }
+
+    uint32_t getDetId() const
+    {
+      return detId;
```

Comment:
It may be better to use TotemRPDetId  directly.
I worry that this will generate unnecessary exposure of the raw detector index to different parts of the code base.
(e.g., someone starts decoding elements from this TotemStructuralVFATId::getDetId directly instead of using TotemRPDetId, the authoritative place to get indices for logical detector elements.